doc/workshops/financial.rst

```diff
@@ -2,4 +2,1065 @@
+
+
+Monte carlo simulation
+~~~~~~~~~~~~~~~~~~~~~~
+
+The Monte Carlo simulation also shows the non-linear correlation between risk and returns (as you can see, sometimes high risk involves only modest profits):
+
+.. image:: images/workshop-1/scatter-plot-sharpe-portfolio-3.png
+   :alt: Portfolio 3 Monte Carlo Sharpe simulation scatter plot
+
+As can be seen, there are points (bottom right) that show a very high volatility and yet have a very low expected return. In this sense, portfolio 1 represents a safer investment because the higher risk is consistently offset by higher returns.
+
+
+Optimal weights
+~~~~~~~~~~~~~~~
+
+.. image:: images/workshop-1/portfolio-3-optimal-weights.png
+   :alt: Portfolio 3 optimal weights by year
+
+The optimal portfolio weights, if calculated annually, suggest that our portfolio should have been quite polarized in some years: the recommendation is to have bought only bitcoins before the start of 2016 and 2019, and only ethereum in 2018. Starting 2017 and 2020, on the other hand, our weights recommended a more balanced investment between bitcoin and ethereum. Litecoin is not recommended by our model.
+
+
+Expected and realized returns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: images/workshop-1/expected-vs-realized-portfolio-3-returns.png
+   :alt: Portfolio 3 expected and realized returns comparison
+
+In this graph we can see that in 2017 and 2020 the earnings obtained would have exceeded the expected earnings (with our calculated weights). In 2019 our model predicted a sharp drop in the portfolio, but in reality the portfolio did not make either annualized gains or losses that year. In contrast, in 2018, our model would have brought us heavy losses, as the value of cryptocurrencies declined sharply that year.
+
+The mean expected return for our portfolio is 0.6972, which is higher than the realized return of 0.4181 that we would have obtained. Having invested in this portfolio over the long term (from 2016 to the present) would have been a very good deal. Due to the high variability, investing in the short term would have been very risky. In terms of gross profits, the realized returns of this portfolio were more than double those of portfolio 1 (0.4181 > 0.1997).
+
+
+
+===========
+Final words
+===========
+
+The mean-variance portfolio (MVP) theory is one of the many tools available to financial analysis. In recent years, machine learning algorithms have even been used to predict the behavior of stock prices more accurately than can be achieved with any standard financial theory.
+
+The examples given during this workshop are not intended to serve as guidelines for you to invest your money. It is only a first step towards learning financial analysis using Python and a scientific IDE.
+
+With the skills learned here, you will be able to approach more complex topics of financial analysis such as those you will find in the references in the next section.
```

Comment:
Maybe leave a"Homework" at the end such as in the previous workshop, maybe ask them to write the code for the third portfolio and compare their results with yours.